gamemodes/terrortown/gamemode/shared/sh_item_module.lua

```diff
@@ -51,6 +60,15 @@ for i = 1, #itemsFolders do
 		end
 	end
 
+	ITEM.CanBuy = ITEM.CanBuy or {}
```

Comment:
Comment to :)

Reply:
This has been removed now, see my other comment :)